Makefile

```diff
@@ -139,7 +142,22 @@ TMPDIR ?= $(if $(shell if [ -d /scratch ] ; then echo true; fi),/scratch/$(USER)
 
 ### Default tag
 help:
-	@echo "Targets:"
+	@echo
+	@echo "This is the primary Makefile for both an archive file based setup"
+	@echo "(i.e. created from daikon-<version>.zip) and for a repository"
+	@echo "based setup (i.e. git cloned from //github.com/codespecs/daikon)."
+	@echo "This is primarily for users who wish to customize or extend DAIKON."
+	@echo
+	@echo "These first two targets are only for the archive version:"
+	@echo " install                  -- build additional tools and kvasir"
+	@echo " install-test             -- simple verification of installtion"
+	@echo
+	@echo "This target is shared; it is for building a modified version"
+	@echo "of the java runtimes needed by the DynComp tool:"
+	@echo " dyncomp-jdk              -- Make file java/dcomp_rt.jar"
+	@echo
+	@echo "The rest of the targets listed below are for a full"
+	@echo "repository based installation. For building:"
```

Comment:
These are also relevant to an archive-based installation if one has made any changes, right?
It might be better to focus more on the use cases.  (I do agree that users generally shouldn't make changes after installing from the archive, but I'm not sure this message is the right place for that information.)